curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-tone-of-a-color.md

```diff
@@ -9,7 +9,7 @@ dashedName: adjust-the-tone-of-a-color
 
 # --description--
 
-The `hsl()` option in CSS also makes it easy to adjust the tone of a color. Mixing white with a pure hue creates a tint of that color, and adding black will make a shade. Alternatively, a tone is produced by adding gray or by both tinting and shading. Recall that the 's' and 'l' of `hsl()` stand for saturation and lightness, respectively. The saturation percent changes the amount of gray and the lightness percent determines how much white or black is in the color. This is useful when you have a base hue you like, but need different variations of it.
+The `hsl()` option in CSS also makes it easy to adjust the tone of a color. Mixing white with a pure hue creates a tint of that color, and adding black will make a shade. Alternatively, a tone is produced by adding gray or by both tinting and shading. Recall that the 's' and 'l' of `hsl()` stand for `saturation` and `lightness`, respectively. The saturation percent changes the amount of gray and the lightness percent determines how much white or black is in the color. This is useful when you have a base hue you like, but need different variations of it.
```

Comment:
The sentence "Recall that the 's' and 'l' of `hsl()` stand for `saturation` and `lightness`, respectively." is a bit tricky. Writing out what an acronym stands for in English is fine for English, but for other languages  I am not sure.  @ojeytonwilliams Do you have some insight on what we should do with acronyms?

Reply:
Yeah, these ones were tricky.

I figured that, when referring to the actual acronym as a definition, they should be code - but in other context I did not.

Reply:
So what have we concluded about the acronym here?

@ojeytonwilliams Final thoughts?

Reply:
My takes were here https://github.com/freeCodeCamp/freeCodeCamp/pull/40838#pullrequestreview-579878299 and here https://github.com/freeCodeCamp/freeCodeCamp/pull/40838#discussion_r567401776

Basically, we'd do more harm than good by imposing too many restrictions on the translators.  If something is code it should be wrapped in code tags and caught by the custom QA.  If it's not code (like saturation, in this case), then it's up to the translators to figure out how best to convey that the 's' of `hsl` refers to 'saturation'. Reason being, that will depend on the specifics of the target language.